blocks/gallery-block/_index.scss

```diff
@@ -10,6 +15,17 @@
 		@include scss.block-components("gallery-image-wrapper");
 		@include scss.block-properties("gallery-image-wrapper");
 	}
+
+	&__button--exit-full-screen {
+		@include scss.block-components("gallery-button-exit-full-screen");
+		@include scss.block-properties("gallery-button-exit-full-screen");
+	}
+
+	&__icon--exit-full-screen {
+		@include scss.block-components("gallery-icon-exit-full-screen");
+		@include scss.block-properties("gallery-icon-exit-full-screen");
+	}
```

Comment:
Same as with the other comment, we should try `"components: { "button": {}, "icon": {} }` first to minimize the amount of tokens we are creating.

Reply:
Comment attended on refactoring.